impexp-client/src/main/java/org/citydb/cli/DeleteCommand.java

```diff
@@ -54,6 +54,9 @@
             description = "Delete mode: ${COMPLETION-CANDIDATES} (default: ${DEFAULT-VALUE}).")
     private Mode mode;
 
+    @CommandLine.Option(names = "--preview", description = "Only print a report of the matched top-level city objects.")
```

Comment:
I think it would be nice if there would also be a one-character option name in addition to `--preview`. Both `-p` and `-P` are already in use by other options, but what about `v`?

Reply:
yes, see here c183afb